tests/phpunit/tests/blocks/registerCoreBlockStyleHandles.php

```diff
@@ -99,6 +99,30 @@ public function test_wp_should_load_separate_core_block_assets_true( $name, $sch
 		}
 	}
 
+	/**
+	 * @ticket 58560
+	 *
+	 * @dataProvider data_block_data
```

Comment:
```suggestion
	 * @dataProvider data_block_data
	 *
	 * @param string $name   The block name.
	 * @param array  $schema The block's schema.
```
Test method parameters should be documented. This could be deferred and addressed for the whole test file though, as it appears none of the test methods have this.